Documentation/ExtensionArchitecture/ComponentsOfTYPO3Extension/CreatingDatabaseModel.rst

```diff
@@ -0,0 +1,35 @@
+.. _creating-database-model:
+
+===================================
+Creating a New Database Model
+===================================
+
+Create SQL Database Schema
+---------------------------
+
+* Add `example-extension/ext_tables.sql`
+* Create database schema
```

Comment:
```suggestion
* Insert your SQL database schema definition into that file
```

(Otherwise people may think that they need to create that on database-level and maybe the file would get automatically created from their database ;) )